closure/goog/graphics/canvasgraphics_test.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<!--
+Copyright 2015 The Closure Library Authors. All Rights Reserved.
+
+Use of this source code is governed by the Apache License, Version 2.0.
+See the COPYING file for details.
+
+-->
+<head>
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<title>Closure Unit Tests - goog.graphics.CanvasGraphics</title>
+<script src="../base.js"></script>
+<script>
+  goog.require('goog.dom');
+  goog.require('goog.graphics.CanvasGraphics');
+  goog.require('goog.graphics.SolidFill');
+  goog.require('goog.graphics.Stroke');
+  goog.require('goog.testing.graphics');
+  goog.require('goog.testing.jsunit');
+</script>
+</head>
+<body>
+<div id="root"> </div>
+
+<script>
+  var graphics;
+
+  function setUp() {
+    graphics = new goog.graphics.CanvasGraphics(100, 100);
+    graphics.createDom();
+    goog.dom.getElement('root').appendChild(graphics.getElement());
+    graphics.enterDocument();
+  }
+
+  function testDrawRemoveRect() {
```

Comment:
We have a new style for tests now. All your .js should be in a separate `_test.js` file, with a single `goog.require` for your test namespace. E.g., see https://github.com/google/closure-library/blob/master/closure/goog/async/debouncer_test.html.  Required test DOM can stay in the .html file.